### PR TITLE
virt-launcher: libvirt and qemu conf cleanup

### PR DESCRIPTION
**Pull request by acardace**
**What this PR does / why we need it**:
Remove duplicated settings we forcibly write in libvirtd.conf and qemu.conf since they're already present in the shipped config files.

Disable libvirt unencrypted TCP socket since we only use the unix socket to communicate with the daemon.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
virt-launcher: disable unencrypted TCP socket for libvirtd.
virt-launcher: 'cgroup_controllers = [ ]' is not injected anymore at runtime and thus is a required setting in the image /etct/libvirt/qemu.conf file.
```

### CODE REVIEW

**Comment by acardace** [comment]
/cc @rmohr @xpivarc @iholder-redhat 
Since we were talking about conf files if you guys could take a look that'd be appreciated, thanks!

File: pkg/virt-launcher/virtwrap/util/libvirt_helper.go [hunk 1]
```diff
@@ -450,12 +450,6 @@ func configureQemuConf(qemuFilename string) (err error) {
 	}
 	defer util.CloseIOAndCheckErr(qemuConf, &err)
 
-	// We are in a container, don't try to stuff qemu inside special cgroups
-	_, err = qemuConf.WriteString("cgroup_controllers = [ ]\n")
```
Comment (xpivarc) [inline pkg/virt-launcher/virtwrap/util/libvirt_helper.go:454]: Now all builds need to include this. Can we make this into release note?

**Review by xpivarc — APPROVED** [review]

File: pkg/virt-launcher/virtwrap/util/libvirt_helper.go (hunk 1 above)
Comment (acardace) [reply pkg/virt-launcher/virtwrap/util/libvirt_helper.go:454]: If you think it brings value yes we can but I don't think we should worry or support custom builds, it's the opposite, they should track upstream and at least use the sensible default settings.

Comment (acardace) [reply pkg/virt-launcher/virtwrap/util/libvirt_helper.go:454]: Release note updated!

**Comment by iholder-redhat** [comment]
Thanks @acardace! Looks great!
/lgtm

File: pkg/virt-launcher/virtwrap/util/libvirt_helper.go [hunk 2]
```diff
@@ -505,16 +499,13 @@ func (l LibvirtWraper) SetupLibvirt() (err error) {
...
 	}
 
-	libvirdDConf, err := os.OpenFile(runtimeLibvirtdConfPath, os.O_APPEND|os.O_WRONLY, 0644)
-	if err != nil {
-		return err
-	}
-	defer util.CloseIOAndCheckErr(libvirdDConf, &err)
-
-	// Let libvirt log to stderr
-	_, err = libvirdDConf.WriteString("log_outputs = \"1:stderr\"\n")
```
Comment (davidvossel) [inline pkg/virt-launcher/virtwrap/util/libvirt_helper.go:515]: why is okay to remove this line and the one regarding cgroups? Where are the duplicate lines coming from in this case?

**Review by davidvossel — COMMENTED** [review]
i get the disabling tcp change, the change related to cgroups and log to stderr is less obvious

File: pkg/virt-launcher/virtwrap/util/libvirt_helper.go (hunk 2 above)
Comment (acardace) [reply pkg/virt-launcher/virtwrap/util/libvirt_helper.go:515]: It's in https://github.com/kubevirt/kubevirt/pull/6145/commits/8cf78f057360dd882a5b7dd40b7bdab357edceb6 commit message, we already have them in the shipped config file that we use to build the images.

**Review by davidvossel — APPROVED** [review]
/approve

File: pkg/virt-launcher/virtwrap/util/libvirt_helper.go (hunk 2 above)
Comment (davidvossel) [reply pkg/virt-launcher/virtwrap/util/libvirt_helper.go:515]: yup, got it now. i see we have these configs in the kubevirt/kubevirt repo even.

**Pull request merged**